Makefile

```diff
@@ -58,14 +58,14 @@ vet:
 
 fmt:
 	@echo "+ $@"
-	@test -z "$$(gofmt -s -l . 2>&1 | grep -v ^vendor/ | tee /dev/stderr)" || \
+	@test -z "$$(gofmt -s -l . 2>&1 | grep -v ^vendor/ | grep -v /vendor/ | tee /dev/stderr)" || \
```

Comment:
I was originally thinking that, but there is no reason we cannot enforce go files in contrib, we just want to ignore anything that has "/vendor/" in its path, I could try and include that into a single regexp, not sure if it would look like something like `grep -v "(^|/)vendor/"`